### PR TITLE
Changes done in Service Helper Junit Classes

### PR DESCRIPTION
**Pull request by aashoo-rajput**

### CODE REVIEW

File: src/test/java/com/imagination/cbs/service/helper/BookingHrApproveHelperTest.java [hunk 1]
```diff
@@ -92,6 +95,31 @@ public void shouldThrowCBSApplicationExceptionWhenBookingAlreadyApprovedOrNotInA
...
+		
+		when(loggedInUserService.isCurrentUserInHRRole()).thenReturn(true);
+		when(bookingSaveHelper.getLatestRevision(booking)).thenReturn(bookingRevision);
+		when(html2PdfService.generateAgreementPdf(bookingRevision)).thenThrow(IllegalStateException.class);
+		when(loggedInUserService.getLoggedInUserDetails()).thenReturn(cbsUser);
+		when(bookingSaveHelper.saveBooking(booking, bookingRevision, 1006L,cbsUser)).thenReturn(booking);
+		
+		bookingHrApproveHelper.hrApprove(booking);
+		
+		verify(loggedInUserService).isCurrentUserInHRRole();
```
Comment (S-U-N-I-L) [inline src/test/java/com/imagination/cbs/service/helper/BookingHrApproveHelperTest.java]: break the verify in separate test cases.

File: src/test/java/com/imagination/cbs/service/helper/CreateBookingHelperTest.java [hunk 2]
```diff
@@ -105,26 +92,20 @@
...
+	public void setUp() throws Exception {
+		
+		 when(loggedInUserService.getLoggedInUserDetails()).thenReturn( new CBSUser("Pappu"));
+		 when(roleRepository.findById(4326L)).thenReturn(Optional.of(new RoleDm()));
+		 when(bookingMapper.toBookingDomainFromBookingDto(Mockito.any(BookingRequest.class))).thenReturn(new Booking());  
+		 when(officeRepository.findById(12345L)).thenReturn(Optional.of(new OfficeDm())); 
+		 when(recruitingRepository.findById(3333L)).thenReturn(Optional.of(new ReasonsForRecruiting()));
+		 when(regionRepository.findById(5555L)).thenReturn(Optional.of(new Region()));
+		 when(siteOptionsRepository.findById(6666L)).thenReturn(Optional.of(new SiteOptions())); 
+		/* when(maconomyService.getMaconomyJobNumberAndDepartmentsDetails(
```
Comment (S-U-N-I-L) [inline src/test/java/com/imagination/cbs/service/helper/CreateBookingHelperTest.java]: comment code .. delete this

File: src/test/java/com/imagination/cbs/service/helper/CreateBookingHelperTest.java [hunk 3]
```diff
@@ -217,7 +209,18 @@ public void shouldCall_findByIdFromSiteOptionsRepository() {
 	}
 	
 	private BookingRequest createBookingRequest(){
-		
+		WorkDaysDto work = new WorkDaysDto();
```
Comment (S-U-N-I-L) [inline src/test/java/com/imagination/cbs/service/helper/CreateBookingHelperTest.java:207]: this method is too big... try to break in multiple private methods.

File: src/test/java/com/imagination/cbs/service/helper/BookingHrApproveHelperTest.java (hunk 1 above)
Comment (aashoo-rajput) [reply src/test/java/com/imagination/cbs/service/helper/BookingHrApproveHelperTest.java]: as per your suggestion. I have implemented in all new Test classes but this an old file, so kept as it is

File: src/test/java/com/imagination/cbs/service/helper/CreateBookingHelperTest.java (hunk 2 above)
Comment (aashoo-rajput) [reply src/test/java/com/imagination/cbs/service/helper/CreateBookingHelperTest.java]: changes done, please check

File: src/test/java/com/imagination/cbs/service/helper/CreateBookingHelperTest.java (hunk 3 above)
Comment (aashoo-rajput) [reply src/test/java/com/imagination/cbs/service/helper/CreateBookingHelperTest.java:207]: changes done, please check

File: src/test/java/com/imagination/cbs/service/helper/BookingHrApproveHelperTest.java (hunk 1 above)
Comment (S-U-N-I-L) [reply src/test/java/com/imagination/cbs/service/helper/BookingHrApproveHelperTest.java]: yes Akshay also told me ... i asked if you can then do it. he is implementing ... check on your end if you can do these changes

Comment (aashoo-rajput) [reply src/test/java/com/imagination/cbs/service/helper/BookingHrApproveHelperTest.java]: ok.. i am doing changes

**Review by S-U-N-I-L — APPROVED** [review]

**Pull request merged**